docs/source/advanced/multi_gpu.rst

```diff
@@ -362,6 +362,9 @@ Distributed Data Parallel Spawn
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 `ddp_spawn` is exactly like `ddp` except that it uses .spawn to start the training processes.
 
+.. note::
+    If you request multiple GPUs or nodes without setting a mode, DDP Spawn will be the default.
```

Comment:
this is direct conflict with the note under the "Distributed Modes" section:
```
.. note::
    If you request multiple GPUs or nodes without setting a mode, DDP will be automatically used.
```

Reply:
@SkafteNicki Should we mention Spawn there itself then?

Reply:
Why not make it use DDP by default?

Reply:
Not sure. Maybe instead of adding a new note, just correct the original note to `DDP spawn` instead of just `DDP`

Reply:
Agree, it should be more visible than a note.

Reply:
Sounds good, will update!

Reply:
Hi, just in case there is confusion here I want to add a comment.
When talking about DDP in Lightning it can mean multiple things. 

a) generally distributed training (using pytorch DDP) 
b) specifically the mode of training known as `accelerator="ddp"` which is different from `accelerator="ddp_spawn"`. 

The differences in the code are subtle, but the consequences are significant. So when we talk about ddp we should make sure that from the context it is clear what we mean. An example of this is what @SkafteNicki mentioned:

> If you request multiple GPUs or nodes without setting a mode, DDP will be automatically used.

This sentence meant to say that we use distributed training (not DP) when using multiple gpus. But the sentence does not specify which ddp plugin specifically (accelerator=ddp or accelerator=ddp_spawn?).

Another example:

> Why not make it use DDP by default?

Is the question "Why not use accelerator=ddp by default?"? If so, the answer is because that would require your program be called as a script, which is not always the case.

Reply:
@awaelchli Yes, and I think we should specify what is the default ddp plugin when using multiple gpus, in the docs.